README.md

```diff
@@ -131,7 +132,8 @@ The following APIs are for developers or using this lib out of the Nginx case.
 ## Nginx APIs
 - **startTimer**, `require("skywalking.client"):startBackendTimer("http://127.0.0.1:8080")`. Start the backend timer. This timer register the metadata and report traces to the backend.
 - **destroyBackendTimer**, `require("skywalking.client"):destroyBackendTimer()`. Stop the timer created by `startBackendTimer`, and clean unreported data.
-- **start**, `require("skywalking.tracer"):start("upstream service", correlation)`. Begin the tracing before the upstream begin. The custom data (table type) can be injected as the second parameter, and then they will be propagated to the downstream service.
+- **start**, `require("skywalking.tracer"):start("upstream service")`. Begin the tracing before the upstream begin.
+- **inject**, `require("skywalking.tracer"):inject(exitSpan, correlation)`. Inject an exit span context and correlation context into carrier. The custom data (table type) can be injected as the second parameter, and then they will be propagated to the downstream service.
```

Comment:
```suggestion
- **inject**, **since 0.7**, `require("skywalking.tracer"):inject(exitSpan, correlation)`. Inject an exit span context and correlation context into carrier. The custom data (table type) can be injected as the second parameter, and then they will be propagated to the downstream service.
```